tests/factories/custom_rule.py

```diff
@@ -0,0 +1,27 @@
+from random import choices, randint
+
+import factory
+from factory import Faker
+from factory.alchemy import SQLAlchemyModelFactory
+
+from checkmate.checker.url.reason import Reason
+from checkmate.models.data import custom_rule
+from checkmate.url import hash_for_rule
+
+REASONS = list(Reason)
+
+
+class CustomRule(SQLAlchemyModelFactory):
+    class Meta:
+        """Metaclass for factoryboy."""
+
```

Comment:
Yep. It's pointless. It would be _very_ nice to have agreement between products about whether we are bumping onto the second line or the first as well. It's very hard to get any muscle memory.

I'm not sure how we could go about it, but I wonder if we can do something to synchronise more of this bumf between projects. Something similar to the update mechanism in the cookie cutter for the libs.